iml-agent/src/cli.rs

```diff
@@ -99,6 +99,20 @@ pub enum PackageCommand {
     },
 }
 
+#[derive(Debug, StructOpt)]
+pub enum KernelModuleCommand {
```

Comment:
Can you add some comments so structopt will print the help text for each command?

Reply:
Done